### PR TITLE
Re-adds lawn mowers.

### PR DESCRIPTION
**Pull request by ghost**
Re-adds lawn mowers.
Currently only admin spawnable.

### CODE REVIEW

**Comment by Spacedong** [comment]
DO NOT LAWN MOWERS NEAR THE CLOWN

**Comment by ghost** [comment]
Oh yeah, it's keyless.

**Comment by SpankMasterson** [comment]
+1

**Comment by yobob591** [comment]
++++++++++11111111111111111111111111111111111

**Comment by Swagile** [comment]
GM + 1 if it passes a code checkeroni

**Comment by GuyonBroadway** [comment]
Re-add the "This is reminder 501 to not drive lawnmowers around the station, or near the clown."

I miss that one.

**Comment by ghost** [comment]
https://raw.githubusercontent.com/HippieStationCode/HippieStation13/master/sound/ambience/ambiencegenlawnmower.ogg
it already exists in the code and is in areas.dm

**Comment by gumner** [comment]
+11

**Comment by zenek123321** [comment]
+1

**Comment by ghost** [comment]
boop boop boop
attention crew, please keep the lawnmower way from stray objects. and the clown.

yes +1

**Comment by zenek123321** [comment]
MERGE NOW

**Comment by tortellinitony** [comment]
make it like the recycler

File: code/game/objects/structures/janicart.dm [hunk 1]
```diff
@@ -321,3 +321,68 @@
...
+				visible_message("<span class='danger'>The [callme] grinds [M] into a fine paste!</span>")
+				M.gib()
+				shake_camera(M, 20, 1)
+				gibbed = 1
+
+	if(gibbed)
+		shake_camera(buckled_mob, 10, 1)
+		playsound(loc, 'sound/effects/mowermovesquish.ogg', 75, 1)
+	else
+		playsound(loc, pick('sound/effects/mowermove1.ogg', 'sound/effects/mowermove2.ogg'), 75, 1)
```
Comment (JohnGinnane) [inline code/game/objects/structures/janicart.dm]: Might be an idea to put these into specsound or something in the definition of the mower

File: code/game/objects/structures/janicart.dm [hunk 2]
```diff
@@ -321,3 +321,68 @@
...
+
+	for(var/obj/effect/spacevine/S in loc)
+		qdel(S)
+
+	if(emagged)
+		for(var/mob/living/carbon/human/M in loc)
+			if(M == buckled_mob)
+				continue
+			if(M.lying)
+				visible_message("<span class='danger'>The [callme] grinds [M] into a fine paste!</span>")
```
Comment (JohnGinnane) [inline code/game/objects/structures/janicart.dm]: Please use M.name

**Review by JohnGinnane — CHANGES_REQUESTED** [review]

**Comment by JohnGinnane** [comment]
Code's pretty good just one minor thing and it's ready

File: code/game/objects/structures/janicart.dm (hunk 2 above)
Comment (ghost) [reply code/game/objects/structures/janicart.dm]: Will do

File: code/game/objects/structures/janicart.dm (hunk 1 above)
Comment (JohnGinnane) [reply code/game/objects/structures/janicart.dm]: Likewise with the above, it's mostly so you can easily change it with view vars :^)

**Comment by MethMedic** [comment]
Lawnmower thunderdome/event when

**Comment by ghost** [comment]
@JohnGinnane finally made the changes requested.

**Review by JohnGinnane — APPROVED** [review]

**Comment by blobbernaut** [comment]
+1

**Comment by Swagile** [comment]
this has 5/6 votes

spank + neko bot +1'd and i + 1'd

requires one GM vote and then it can be merged

**Comment by ghost** [comment]
one headmin +1 is 3/6, swagile. two headmins giving their +1 means it can be merged.

**Comment by Swagile** [comment]
im retarded and this actually only needed 2 heads, im just the middle man out serving as the gatekeeper to the real power behind the mans

tl;dr this can be merged

**Comment by JohnGinnane** [comment]
Merging
